### ISSUE TITLE
⚙️ Module "blowfish" is not compatible with this Hugo version

### ISSUE THREAD

**Issue by tfunken** [nunocoracao/blowfish#909]
**Describe the bug**

Hugo shows the error:

WARN Module "blowfish" is not compatible with this Hugo version; run "hugo mod graph" for more information.

**Hugo & Blowfish versions**
Hugo: 0.117
Blowfish: v2.38.3

**Comment by olliefr** [comment]
I get the same message but my (admittedly simple) website builds OK and I haven't noticed any issues apart from that message being displayed. 

I installed Hugo from snap on Ubuntu 22.04.3 LTS where the `stable` channel currently is at `v0.117.0` and there is not much freedom to install an older version.

My Blowfish version is `v2.38.3`.

What I did to make the warning message go away:

Replace `max = "0.115.4"` in `themes/blowfish/config.toml` with `max = "0.117.0"`
